services/route_manager/src/main/resources/application.properties

```diff
@@ -2,6 +2,8 @@ server.port=9003
 #####Microservice url configuration######
 microservices.subnet.service.url=http://localhost:9002
 microservices.vpc.service.url=http://localhost:9001
+microservices.dpm.service.url=http://localhost:9010
```

Comment:
For the new configurations in RM and SM, don't forget to add them in K8s yaml file :-)